### PR TITLE
fix: Inline how-to-guide

### PR DESCRIPTION
**Pull request by harshilp24**
## Checklist
I have:
- [ ] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [ ] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [ ] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/reference/widgets/table/inline-editing.md [hunk 1]
```diff
@@ -1,134 +1,109 @@
...
 
-This page describes how to add and edit Table data directly from the UI through inline editing.
 
 <VideoEmbed host="youtube" videoId="eIecDfvSOsU" title="" caption=""/>
 
 
-## Edit cells
+## Prerequisites
+
+* A [Table widget](/reference/widgets/table) connected to a query that holds the data you want to edit and update.
```
Comment (danciaclara) [inline website/docs/reference/widgets/table/inline-editing.md:11]:
```suggestion
* A [Table widget](/reference/widgets/table) connected to a query with the data you want to edit and update.
```

File: website/docs/reference/widgets/table/inline-editing.md [hunk 2]
```diff
@@ -1,134 +1,109 @@
...
-5. Set the Table widget's **onSave** event to run the update query, and **onSuccess** callback to run `fetchData` query.
-
-As an alternative you can configure the **onSubmit** event in each [column's settings](/reference/widgets/table/column-settings) to run a query that saves the new data. The **onSubmit** event takes place whenever the user clicks away from the edited cell, or presses the Enter key within it. 
-
+</dd>
 
+3). Click on the gear icon ⚙️ next to the **Save/Discard** column in the property pane.
 
-### Update multiple rows
+4). Set the **onSave** event to run the update query, and **onSuccess** callback to trigger a fetch query that refreshes the data.
```
Comment (danciaclara) [inline website/docs/reference/widgets/table/inline-editing.md]:
```suggestion
4). Set the **onSave** event to run the update query, and the  **onSuccess** callback to trigger the fetch query that refreshes the table data with the updated information.
```

File: website/docs/reference/widgets/table/inline-editing.md [hunk 3]
```diff
@@ -1,134 +1,109 @@
...
+ 
+As an alternative you can configure the **onSubmit** event in each [column's settings](/reference/widgets/table/column-settings) to run a query that saves the new data. The **onSubmit** event takes place whenever the user clicks away from the edited cell, or presses the Enter key within it.
```
Comment (danciaclara) [inline website/docs/reference/widgets/table/inline-editing.md:56]:
```suggestion
Alternatively, you can configure the **onSubmit** event for each [Column](/reference/widgets/table/column-settings) to run a query that saves the new data. The **onSubmit** event is triggered when the user clicks away from the edited cell or presses the Enter key within it. 
```

File: website/docs/reference/widgets/table/inline-editing.md (hunk 2 above)
Comment (danciaclara) [reply website/docs/reference/widgets/table/inline-editing.md]: repeat this wherever the same statement is being used

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/reference/widgets/table/inline-editing.md (hunk 2 above)
Comment (harshilp24) [reply website/docs/reference/widgets/table/inline-editing.md]: done

**Review by danciaclara — APPROVED** [review]

**Pull request merged**